package.json

```diff
@@ -38,6 +38,7 @@
     "data-uri-to-buffer": "0.0.4",
     "datauri": "^1.0.0-alpha.5",
     "image-size": "^0.5.0",
+    "jimp": "^0.2.24",
```

Comment:
Add this to the third-party section of [LICENSE.md](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/LICENSE.md).  In general, can we please update that section to match the libraries here in `dependencies`?